src/components/Commands.tsx

```diff
@@ -48,8 +41,63 @@ const Command = (props: any) => {
 }
 
 const Commands = () => {
+  const [loading, setLoading] = useState(false)
+  const [isAsanaTaskFormVisible, setIsAsanaTaskFormVisible] = useState(false);
+  const [asanaFormRef, setAsanaFormRef] = useState()
+  const [credentials] = useAsanaCredentials();
+  const { createTask } = useCreateAsanaTask();
+
+  const isAsanaAuthorized = !!credentials.access_token
+  const commands = [{
+    name: 'Create Task',
+    icon: sunsamaLogo,
+    command: () => message.success('Creating a task is coming soon')
+  }, {
+    name: isAsanaAuthorized ? 'Create Asana Task' : 'Connect to Asana',
+    icon: asanaLogo,
+    command: () => {
+      if (!isAsanaAuthorized) {
+        window.location.replace(createClient().app.asanaAuthorizeUrl())
+      } else {
+        setIsAsanaTaskFormVisible(true)
+      }
+    }
+  }]
```

Comment:
This is a really nice touch in the case where a user has yet to add the Asana integration. Great work!